0xProject/0x-monorepo
# Unmount Instant when closing

## Description

See https://github.com/0xProject/0x-monorepo/issues/1694

Unmounts component when Instant is closed. 

Also, provides public `unmount` function for users who render Instant "inline".  For example, MyCrypto  can call `zeroExInstant.unmount` from a `componentWillUnmount` [here](https://github.com/MyCryptoHQ/MyCrypto/blob/eb59b9942491b1a0826395a4f988a1e08ae11256/common/v2/features/BuyAndExchange/ZeroEx/ZeroEx.tsx#L17).

## Testing instructions

### Repro
- Checkout master
- Select token and type amount
- Close instant
- Notice instant is still polling SRA endpoint

### Test fix
- Checkout this branch
- Notice that bug is fixed: when Instant is closed, polling stops.

## Types of changes









## Checklist:
